analyzers/tests/SonarAnalyzer.Test/Rules/TestMethodShouldContainAssertionTest.cs

```diff
@@ -26,7 +26,7 @@ namespace SonarAnalyzer.Test.Rules
     [TestClass]
     public class TestMethodShouldContainAssertionTest
     {
-        private const string Latest = Constants.NuGetLatestVersion; // Rename only
+        private const string Latest = SonarAnalyzer.TestFramework.Common.Constants.NuGetLatestVersion; // Rename only
```

Comment:
Why not import the namespace? Just curious.

Reply:
Because I forgot to remove this one. There was the collision with `SonarAnalyzer.Constants` namespace that I renamed and I forgot to undo this prototyping